bundles/org.openhab.binding.connectedcar/README.md

```diff
@@ -0,0 +1,585 @@
+|              | mileage                     | Number:Length        | yes     | Distance for this trip.                                                                         |
+|              | startMileage                | Number:Length        | yes     | Start Milage for the trip                                                                       |
+|              | overallMileage              | Number:Length        | yes     | Overall milage after this trip                                                                  |
+|              | avgSpeed                    | Number:Speed         | yes     | Average Speed for this trip                                                                     |
+|              | avgElectricConsumption      | Number:Energy        | yes     | Electrical consumption during the trip                                                          |
+|              | avgFuelConsumption          | Number:Volume        | yes     | Average fuel consumption for this trip                                                          |
+| tripLong1    | timestamp                   | DateTime             | yes     | Trip time                                                                                       |
+|              | mileage                     | Number:Length        | yes     | Distance for this trip.                                                                         |
+|              | startMileage                | Number:Length        | yes     | Start Milage for the trip                                                                       |
+|              | overallMileage              | Number:Length        | yes     | Overall milage after this trip                                                                  |
+|              | avgSpeed                    | Number:Speed         | yes     | Average Speed for this trip                                                                     |
+|              | avgElectricConsumption      | Number:Energy        | yes     | Electrical consumption during the trip                                                          |
+|              | avgFuelConsumption          | Number:Volume        | yes     | Average fuel consumption for this trip                                                          |
+| rluHistory1  | rluOperation                | String               | yes     | Action type: lock/unlock                                                                        |
+|              | rluTimestamp                | DateTime             | yes     | Timestamp when the Lock/Unlock action was initiated                                             |
+|              | rluResult                   | String               | yes     | Action result: 1=ok                                                                             |
```

Comment:
You could make this Number and use the state option. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#state-description